frontend/src/store/modules/LookupStore.ts

```diff
@@ -90,6 +91,9 @@ export default {
     sobonOrientierungswertJahr: (state: LookupState): Array<LookupEntryDto> => {
       return state.sobonOrientierungswertJahr;
     },
+    sobonOrientierungswertJahrWithoutStandortabfrage: (state: LookupState): Array<LookupEntryDto> => {
```

Comment:
Da die meisten Variablen auch eher eine Zusammensetzung deutscher Wörter sind, würde ich das hier ebenfalls so machen.

Reply:
In den Coding-Guidelines ist definiert, dass fachliche Benennungen in Deutsch und der Rest in Englisch geschrieben wird.